frontend/src/pages/NewRunV2.tsx

```diff
@@ -81,89 +86,25 @@ function NewRunV2(props: NewRunV2Props) {
-        throw new Error('No Run ID');
-      }
-      const originalRun = Apis.runServiceApi.getRun(originalRunId);
-      return originalRun;
-    },
-    { enabled: !!originalRunId, staleTime: Infinity },
-  );
+  const templateString = props.templateString;
+  const isTemplatePullSuccess = templateString !== '';
+  const apiResourceRefFromRun = apiRun ? apiRun.run?.resource_references : undefined;
 
-  let templateStringFromRunId;
-  if (apiRun) {
-    templateStringFromRunId = apiRun.run?.pipeline_spec?.pipeline_manifest;
-  }
-  const isTemplatePullSuccessFromRun = templateStringFromRunId !== undefined ? true : false;
-  const apiResourceRefFromRun = apiRun !== undefined ? apiRun?.run?.resource_references : undefined;
-
-  // Retrieve Pipeline Detail using pipeline ID and pipeline version ID from backend.
-  // It validates that the pipeline entity indeed exists with pipeline ID.
-  // TODO(zijianjoy): Need to implement the feature to choose pipeline ID and pipeline version ID from this page.
-  // TODO(zijianjoy): Need to show error if pipeline fetch failed to show up.
-  const { data: apiPipeline } = useQuery<ApiPipeline, Error>(
-    ['ApiPipeline', pipelineId],
-    () => {
-      if (!pipelineId) {
-        throw new Error('Pipeline ID is missing');
-      }
-      return Apis.pipelineServiceApi.getPipeline(pipelineId);
-    },
-    { enabled: !!pipelineId, staleTime: Infinity },
-  );
-  const { data: apiPipelineVersion } = useQuery<ApiPipelineVersion, Error>(
-    ['ApiPipelineVersion', apiPipeline, pipelineVersionIdParam],
-    () => {
-      const pipelineVersionId = pipelineVersionIdParam || apiPipeline?.default_version?.id;
-      if (!pipelineVersionId) {
-        throw new Error('Pipeline Version ID is missing');
-      }
-      return Apis.pipelineServiceApi.getPipelineVersion(pipelineVersionId);
-    },
-    { enabled: !!apiPipeline, staleTime: Infinity },
-  );
-  const {
-    isSuccess: isTemplatePullSuccessFromPipeline,
-    data: templateStringFromPipelineId,
-  } = useQuery<string, Error>(
-    ['ApiPipelineVersionTemplate', apiPipeline, pipelineVersionIdParam],
-    async () => {
-      const pipelineVersionId = apiPipelineVersion?.id;
-      if (!pipelineVersionId) {
-        return '';
-      }
-      const template = await Apis.pipelineServiceApi.getPipelineVersionTemplate(pipelineVersionId);
-      return template?.template || '';
-    },
-    { enabled: !!apiPipelineVersion, staleTime: Infinity },
-  );
-
-  const templateString =
-    templateStringFromRunId !== undefined ? templateStringFromRunId : templateStringFromPipelineId;
-  const isTemplatePullSuccess = isTemplatePullSuccessFromRun || isTemplatePullSuccessFromPipeline;
   const isRecurringRun = urlParser.get(QUERY_PARAMS.isRecurring) === '1';
-  const titleVerb = originalRunId ? 'Clone' : 'Start';
-  const titleAdjective = originalRunId ? '' : 'new';
+  const titleVerb = apiRun ? 'Clone' : 'Start';
+  const titleAdjective = apiRun ? '' : 'new';
```

Comment:
Why do we change from `apiRun ` to `originalRunId `?

Reply:
I am guess that you're curious why change from `originalRunId` to `apiRun`. I would say the existence of `originalRunId` lead to the existence of `apiRun`. The main reason for the change is to make all criteria to check if it's `creating new run from pipeline` or `cloning existing run from run`. I use `apiRun` to determine if we show pipeline and pipeline version section, so I tend to use same condition to check here.

Reply:
Thanks for correction. But `apiRun` can be empty, however `originalRunId` determines whether it is `using run Id` or `using pipeline id`. Since these titles are static, we can use `originalRunId` instead.